setup_chef_cookbooks.sh

```diff
@@ -53,25 +53,5 @@ no_proxy no_proxy_string
 ENV['GIT_SSL_NO_VERIFY'] = 'true'
 File.umask(0007)
 EOF
-cd cookbooks
 
-# allow versions on cookbooks via "cookbook version"
-for cookbook in "apt 2.4.0" python "build-essential 3.2.0" ubuntu cron "chef-client 4.2.4" "chef-vault 1.3.0" "ntp 1.10.1" yum "logrotate 1.9.2" yum-epel "sysctl 0.7.5" chef_handler 7-zip seven_zip "windows 1.36.6" ark sudo ulimit pam "ohai 3.0.1" "poise 1.0.12" graphite_handler java "maven 2.1.1" "krb5 2.0.0" homebrew; do
-  # unless the proxy was defined this knife config will be the same as the one generated above
-  if [[ ! -d ${cookbook% *} ]]; then
-    # 7-zip has been deprecated but recipies still depend on it, will force download
-    if [[ "$cookbook" == "7-zip" || "$cookbook" == "python" ]]; then
-      knife cookbook site download $cookbook --config ../.chef/knife.rb --force
-    else
-      knife cookbook site download $cookbook --config ../.chef/knife.rb
-    fi
-    tar zxf ${cookbook% *}*.tar.gz
-    rm ${cookbook% *}*.tar.gz
-  fi
-done
-if [[ ! -d kafka ]]; then
-  git clone https://github.com/mthssdrbrg/kafka-cookbook.git kafka
-fi
-[[ -d jmxtrans ]] || git clone https://github.com/bijugs/jmxtrans-cookbook.git -b ver-2.0 jmxtrans
-[[ -d cobblerd ]] || git clone https://github.com/bloomberg/cobbler-cookbook.git cobblerd
-[[ -d bfd ]] || git clone https://github.com/bloomberg/openbfdd-cookbook.git bfd
+/opt/chefdk/bin/berks vendor ./cookbooks
```

Comment:
Is this idempotent and does this work in disconnected clusters (as `build_bins.sh` is run with no Internet access).